### ISSUE TITLE
Help !

### ISSUE THREAD

**Issue by foufara4853** [mpld3/mpld3#423]
Hi,
this might not be the right place to ask ?
I want a figure with on the left a scatter plot, and on the right a plot of serveral curves.
Each curve on the right corresponds to one and only one point on the left.
I would like the corresponding point to highlight when I mouseover a line. 
Is that possible ? 
Can anyone give me an example of that ?

Thanks a lot !
Raphaël

**Comment by etgalloway** [comment]
It is probably better to ask questions on StackOveflow.

This example displays a curve when you hover over a data point: http://mpld3.github.io/examples/custom_plugin.html

**Comment by foufara4853** [comment]
Yes Thank you. I saw that example. But I don't know anything about js. I thought someone could give me a very simple example I could adapt. I'll try StackOverflow.
Thank you for your answer.
